src/main/java/choaticbot/actions/Mark.java

```diff
@@ -0,0 +1,19 @@
+package choaticbot.actions;
+
+import choaticbot.tasks.TaskList;
+
+public class Mark extends Action {
+    public String details;
```

Comment:
Perhaps a more specific name would better signal what details this string contains and also avoid the confusion of it being a list / array ?